src/domain/accounts/index.ts

```diff
@@ -0,0 +1,9 @@
+export const AccountLevel = {
+  One: 1,
+  Two: 2,
+} as const
```

Comment:
If we implement the type in the `.d.ts` file, this can probably change. Also, would this be a good candidate for a typescript `enum`?

```suggestion
export const AccountLevel: AccountLevelType = {
  One: 1,
  Two: 2,
}
```